bitnami/grafana-tempo/templates/compactor/deployment.yaml

```diff
@@ -0,0 +1,159 @@
+{{- if .Values.compactor.enabled }}
+apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
+kind: Deployment
+metadata:
+  name: {{ template "common.names.fullname" . }}-compactor
```

Comment:
have you considered to include a function like grafana.compactor.fullname and to use printf instead of include directly the "-compactor"? You can reuse it in several part of the chart. This is a suggestion, it is ok for me.

(same with the other files)

ex: https://github.com/bitnami/charts/blob/e3582cb23180e1fdbc051cfede3dd74437c565b9/bitnami/cert-manager/templates/_helpers.tpl#L34